packages/gas-station/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@turnkey/gas-station",
+  "version": "0.1.0",
+  "description": "Turnkey Gas Station SDK for EIP-7702 delegated execution",
+  "main": "dist/index.js",
+  "module": "dist/index.mjs",
+  "types": "dist/index.d.ts",
+  "exports": {
+    ".": {
+      "import": "./dist/index.mjs",
+      "require": "./dist/index.js",
+      "types": "./dist/index.d.ts"
+    }
+  },
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "rollup -c ../../rollup.config.base.mjs",
+    "typecheck": "tsc --noEmit",
+    "clean": "rimraf ./dist ./.cache"
+  },
+  "keywords": [
+    "turnkey",
+    "gas-station",
+    "eip-7702",
+    "delegated-execution",
+    "paymaster",
+    "ethereum",
+    "viem"
+  ],
+  "dependencies": {
+    "viem": "^2.34.0"
+  },
+  "devDependencies": {
+    "@types/node": "^24.6.2",
+    "typescript": "^5.3.0"
+  },
+  "peerDependencies": {
+    "viem": "^2.0.0"
```

Comment:
big diff here between ^2.0.0 and the ^2.34.0 pinned above. should we require something > 2.0.0?

cc @toctou-security -- shall we hard-pin some versions here?